api/jest.utils.ts

```diff
@@ -7,10 +7,72 @@ declare global {
   var fastifyTestInstance: Awaited<ReturnType<typeof build>> | undefined;
 }
 
+// TODO: remove this function and use superRequest instead
```

Comment:
Why not do this now?

Reply:
I was vaguely thinking that I should keep this focused, but you're right.